resources/monitoring/charts/grafana/templates/clusterrole.yaml

```diff
@@ -3,10 +3,7 @@ kind: ClusterRole
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
   labels:
-    app: {{ template "grafana.name" . }}
```

Comment:
where are all this label defined/configured?

Reply:
on helpers file: https://github.com/kyma-project/kyma/pull/7927/files#diff-5c9321b7eb26a1ca1f487014857d878dR56